<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="edit_panel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#edit_panel{
background-color:#252525;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>241</width>
       <height>232</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="start_btn">
            <property name="styleSheet">
             <string notr="true">background-color:#303030;
color: #FAFAFA;

QPushButton::disabled{
	background-color:#252525;
}</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_btn">
            <property name="styleSheet">
             <string notr="true">background-color:#303030;
color: #FAFAFA;

QPushButton::disabled{
	background-color:#252525;
}</string>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="apply_btn">
            <property name="styleSheet">
             <string notr="true">background-color:#303030;
color: #FAFAFA;

QPushButton::disabled{
	background-color:#252525;
}</string>
            </property>
            <property name="text">
             <string>Apply</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="world_name_label">
              <property name="styleSheet">
               <string notr="true">color:#FAFAFA;</string>
              </property>
              <property name="text">
               <string>World name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="world_name_edit">
              <property name="styleSheet">
               <string notr="true">#world_name_edit{
background-color: #333333;
border-radius: 0px;
color:#FAFAFA;
text-align: center;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="scaling_label">
              <property name="styleSheet">
               <string notr="true">color:#FAFAFA;</string>
              </property>
              <property name="text">
               <string>Scaling</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="scaling_edit">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="speed_label">
              <property name="styleSheet">
               <string notr="true">color:#FAFAFA;</string>
              </property>
              <property name="text">
               <string>Speed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="speed_edit">
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="speed_text_label">
              <property name="styleSheet">
               <string notr="true">color: #FAFAFA;
</string>
              </property>
              <property name="text">
               <string>Speed:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="speed_value_label">
              <property name="styleSheet">
               <string notr="true">color: #FAFAFA;
</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="dots_label">
              <property name="styleSheet">
               <string notr="true">color: #FAFAFA;
</string>
              </property>
              <property name="text">
               <string>Dots:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dots_value_label">
              <property name="styleSheet">
               <string notr="true">color: #FAFAFA;
</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="russian_lang">
            <property name="styleSheet">
             <string notr="true">color:#909090</string>
            </property>
            <property name="text">
             <string>Русский</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="english_lang">
            <property name="styleSheet">
             <string notr="true">color:#909090</string>
            </property>
            <property name="text">
             <string>English</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="styleSheet">
           <string notr="true">color:#909090;</string>
          </property>
          <property name="text">
           <string>Show field</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menubar{
background-color: #202020;
color: #FAFAFA;
}</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">#statusbar{
background-color:#00bfff;
color: #FAFAFA;
font-weight: bold;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
